tests/utils.go

```diff
@@ -5003,3 +5004,14 @@ func IsLauncherCapabilityValid(capability k8sv1.Capability) bool {
 func RetValue(retcode string) string {
 	return "\n" + retcode + CRLF + ".*" + PromptExpression
 }
+
+func CheckDirectIOFlag(path string) bool {
+        // check if fs where disk.img file is located or block device
+        // support direct i/o
+        f, err := os.OpenFile(path, syscall.O_RDONLY|syscall.O_DIRECT, 0)
```

Comment:
I thought this call would be able to tell if the underlying filesystem supports DIRECTIO and accordingly the test verify the cache mode value. But, since this test is run from outside the clusteer, the "os.OpenFile" system call basically runs on the machine that is running the test and not the pod hosting the VM. So I will have to rethink this approach and modify the patch accordingly.

Reply:
You can execute to launcher pod of the vm to find this out.